src/metric_config.cpp

```diff
@@ -182,4 +184,37 @@ void MetricConfig::setAllMetricsTo(bool enabled) {
     requestFailRestModelReady = enabled;
 }
 
+Status MetricConfig::loadSettings(bool isEnabled, std::string metricsList) {
+    using namespace rapidjson;
+    Document document;
+    document.SetObject();
+    Document::AllocatorType& allocator = document.GetAllocator();
+
+    Value metrics(kObjectType);
+    {
```

Comment:
Why do we need scope?

Reply:
Pollutes the code without real need to have it here

Reply:
Fixed.